remo/profiles/models.py

```diff
@@ -110,9 +110,9 @@ class UserProfile(caching.base.CachingMixin, models.Model):
         max_length=DISPLAY_NAME_MAX_LENGTH, blank=True, default='',
         unique=True,
         validators=[
-            RegexValidator(regex=r'("")|(^[A-Za-z0-9_]+$)',
-                           message='Please only A-Z characters, numbers and '
-                                   'underscores.')])
+            RegexValidator(regex=r'^[\w._\-]+$',
```

Comment:
This is not true if `.` is inside `[]`
https://docs.python.org/2/library/re.html